### PR TITLE
Completed javascript-objects exercise. - Henry Ding

### PR DESCRIPTION
**Pull request by henryrding**
image

### CODE REVIEW

File: javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+console.log('value of student.livesInIrvine:', student.livesInIrvine);
+console.log('value of student.previousOccupation:', student.previousOccupation);
+
+var vehicle = {
+  make: 'Honda',
+  model: 'Accord',
+  year: 2013
+};
+
+vehicle.color = 'silver';
```
Comment (shawnkost) [inline javascript-objects/main.js:21]: You should be using bracket notation here

File: javascript-objects/main.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+vehicle.isConvertible = false;
```
Comment (shawnkost) [inline javascript-objects/main.js:22]: See note above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Henry! A couple changes requested below. You should be using bracket notation for the vehicle object

**Comment by henryrding** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**